app/assets/stylesheets/cart.css.scss

```diff
@@ -1,7 +1,9 @@
 .cart {
-  background:white;
+  background-image: 'texture.png';
+    background: oldlace;
```

Comment:
Color `oldlace` should be written in hexadecimal form as `#fdf5e6`<br>Color literals like `oldlace` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces